beta/project-griffon/using-project-griffon/adobe-analytics-and-project-griffon.md

```diff
@@ -6,7 +6,21 @@ The integration with Adobe Analytics provides a richer view of SDK events to use
 
 ![](../../../.gitbook/assets/aa-loop.gif)
 
-If the SDK made a network request with Adobe Analytics \(queued\), and if post-processing information is returned from Adobe Analytics about that event \(validated\), the status tells you when an SDK event is generated \(processed\).
+### Post-processed status
+
+If the SDK made a network request with Adobe Analytics, and if post-processing information is returned from Adobe Analytics about that event, the status tells you when an SDK event is generated.
```

Comment:
After the SDK made a network request with Adobe Analytics, this shows the post-processing information status about that event when returned from Adobe Analytics.

@calebk1 Can you help to review this sentence?

Reply:
I'd go with something more like "After the SDK makes a network request with Adobe Analytics, the status will tell you when the SDK event is generated if post-processing information is returned."

Reply:
Sorry for the confusion with the wording, I wanted to clarify what the status means. The status actually indicates what is happening to the network request sent from Griffon to get the post-processing information after the SDK makes a network request with Analytics, rather than when the SDK event is generated. 

Would this be a better option? `After the SDK makes a network request with Adobe Analytics, the status will tell you if the Griffon was able to retrieve the post-processing information for the Analytics request`

Reply:
That sounds a bit better! I'd go with "After the SDK makes a network request with Adobe Analytics, the status will tell you if Griffon was able to retrieve the post-processing information for the Adobe Analytics request."